code/lib/core-server/src/presets/common-preset.ts

```diff
@@ -185,15 +185,21 @@ export const previewAnnotations = async (base: any, options: Options) => {
   return [...config, ...base];
 };
 
+const fastBuildFeatures = (value: boolean) => ({
```

Comment:
I think we should also in our codebase start calling it the "testBuild" instead of the "fastBuild"

Reply:
That's tricky, because then it sounds like a build that is being tested (as in, CI).  In this case, it would be `testBuildFeatures`, which can be read as "test buildFeatures" or "testBuild features".  I don't have a great suggestion, but `testingBuild` comes to mind as slightly improved, since it's a build used for testing.  Just thought I'd throw that out there if we're bikeshedding names.